src/Components/Web/src/Forms/InputFile/BrowserFileExtensions.cs

```diff
@@ -28,10 +28,33 @@ public static class BrowserFileExtensions
         /// <param name="maxHeight">The maximum image height</param>
         /// <returns>A <see cref="ValueTask"/> representing the completion of the operation.</returns>
         public static ValueTask<IBrowserFile> RequestImageFileAsync(this IBrowserFile browserFile, string format, int maxWidth, int maxHeight)
+        {
+            return browserFile.RequestImageFileAsync(format, maxWidth, maxHeight, null);
+        }
+        
+        /// <summary>
+        /// Attempts to convert the current image file to a new one of the specified file type and maximum file dimensions.
+        /// <para>
+        /// Caution: there is no guarantee that the file will be converted, or will even be a valid image file at all, either
+        /// before or after conversion. The conversion is requested within the browser before it is transferred to .NET
+        /// code, so the resulting data should be treated as untrusted.
+        /// </para>
+        /// </summary>
+        /// <remarks>
+        /// The image will be scaled to fit the specified dimensions while preserving the original aspect ratio.
+        /// The quality parameter is not supported on all browsers, especially mobile browsers.
+        /// </remarks>
+        /// <param name="browserFile">The <see cref="IBrowserFile"/> to convert to a new image file.</param>
+        /// <param name="format">The new image format.</param>
+        /// <param name="maxWidth">The maximum image width.</param>
+        /// <param name="maxHeight">The maximum image height</param>
+        /// <param name="quality">The quality of the image</param>
```

Comment:
Could we extend the comment to give a hint about what is the reasonable range of values? For example, does it go zero to one, or zero to one hundred (a percentage), or zero to infinity, or do the values have some other meaning? What nonempty value is equivalent to the default if you leave it unspecified?

Reply:
Good idea - I added the text from MDN.